doc/conf.py

```diff
@@ -76,7 +76,8 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = ['auto_examples/index.rst', '_build', 'Thumbs.db',
```

Comment:
why do we have to exclude the index.tst from auto_examples?

Reply:
Because otherwise Sphinx would try to process it automatically, and since we're manually `include`ing it in `use.rst` again, it would throw a warning about the same ID used twice. Adding it to the `exclude_patterns` seemed like the easiest way for me to fix it, but I'm not a Sphinx expert…

Reply:
okay, that sounds reasonable. thanks.